runner/local/simple.go

```diff
@@ -62,11 +62,28 @@ func (r *simpleRunner) Run(cmd *runner.Command) (runner.ProcessStatus, error) {
 		return runner.ProcessStatus{}, fmt.Errorf(runner.RunnerBusyMsg)
 	}
 
+	// Redirect command output to the stdout/stderr Output vars. Outputs shall be closed in run().
```

Comment:
Moving this is a bit of a layering violation... Is there a reason to do it here? Just to make it simpler to not have to wait?